lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -818,26 +819,147 @@ namespace Js
             return scriptContext->GetLibrary()->GetUndefined();
         }
 
-#ifdef INTL_WINGLOB
+        DynamicObject *options = DynamicObject::FromVar(args.Values[1]);
+
+        HRESULT hr = S_OK;
+        Var propertyValue = nullptr; // set by the GetTypedPropertyBuiltInFrom macro
```

Comment:
Not sure if this is a macro that you made or not but I found it super confusing to reason about, even with this comment.

Reply:
Yes, I agree. I did not write this macro and the WinGlob code has super confusing logic surrounding it, so I didn't want to rewrite or change anything there at this time. I think this is worth a refactoring at some point.